proseco/core.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, data=None, print_log=False, **kwargs):
             self._default_formats[name] = '.2f'
         for name in ('ra', 'dec', 'RA_PMCORR', 'DEC_PMCORR'):
             self._default_formats[name] = '.6f'
+        self.thumbs_up = False
```

Comment:
Looks l don't really have booleans on the matlab side and the python interface won't pass a numpy.bool type so I end up converting to int anyway.  Should probably just store these as ints on the python side.

Reply:
As always, leave a vertical line of space between entirely unrelated blocks (or lines) of code.

Reply:
Add a `thumbs_up` property which sets onto the same-named meta key.
```
@thumbs_up.setter
def thumbs_up(self, val):
    self.meta['thumbs_up'] = int(val)  # Coerce to int for interface with Matlab
```

Reply:
I thought thumbs_up already was a property set to the same-named meta key?  Wouldn't this just be a change to for the type?

Reply:
Indeed...